src/definitions.ts

```diff
@@ -761,6 +773,19 @@ export interface AuthDependencies {
   storage: IStorage<string>;
 }
 
+
+export interface IGoogleData {
+  webClientId: string;
+}
+
+export interface IFacebookNativeAuth {
+  authenticate(fields: string[]): Promise<any>;
```

Comment:
I think we should have fields be a config value if anything, and then add the fields _we_ need (email, profile_picture, etc)

It's not like these fields have any effect on what we store on the server, right? Does that mean the values for the fields only come back here? Maybe I'm misunderstanding.